scripts/Update-DotnetVersion.ps1

```diff
@@ -0,0 +1,35 @@
+param ( [string]$version = "6.0")
+
+$encoding = [System.Text.UTF8Encoding]::new($false)
+
+$pth = Split-Path $MyInvocation.MyCommand.Path -Parent
+$uri = "https://raw.githubusercontent.com/dotnet/core/main/release-notes/$version/releases.json"
+
+#get version info
+$R = Invoke-WebRequest -Uri $uri -ContentType "application/json" -Method Get -UseBasicParsing
+$versionResult = $R.Content | Out-String | ConvertFrom-Json 
+Write-Output "Dotnet version info"
+$versionResult | Format-List -Property channel-version, latest-release, latest-release-date, latest-runtime, latest-sdk, support-phase, eol-date
+
+#update global.json file
+$globalJson = [System.IO.Path]::Combine( $pth, "..\global.json")
+$globalJson = Resolve-Path $globalJson
+$json = Get-Content $globalJson | ConvertFrom-Json
+Write-Output "Updating SDK version from $($json.sdk.version) to $($versionResult.'latest-sdk')"
+$json.sdk.version = $versionResult.'latest-sdk'
+$json | ConvertTo-Json  | Out-File -Encoding utf8 $globalJson
+
+#update RepoLayout.props
+$propsPth = [System.IO.Path]::Combine( $pth, "RepoLayout.props")
+[xml]$props = Get-Content -Path $propsPth
+$nd = $props.SelectSingleNode("/Project/PropertyGroup/RuntimeFrameworkVersion")
+Write-Output "Updating Runtime version from $($nd.'#text') to $($versionResult.'latest-runtime')"
+$nd.'#text' = $versionResult.'latest-runtime'
+$settings = [System.Xml.XmlWriterSettings]@{
+  Encoding = $encoding
+  Indent   = $true    
+}
+$writer = [System.Xml.XmlWriter]::Create($propsPth, $settings)
+$nd.OwnerDocument.Save($writer)
+$writer.Dispose()
```

Comment:
Not saying that this is wrong, but any reason not to use the PowerShell's built-in [`Export-Clixml`](https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/export-clixml)?

Reply:
I'll replace.  Thanks.

Reply:
@RussKie Digging into, it really messes with format and escapes xml.  This code the way I have it only makes the change and keeps indention and updates the necessary line.  I tried several different inputs to see if I could get it to behave with Export-Clixml

![image](https://user-images.githubusercontent.com/3629489/189567907-ebf12b0b-7614-4305-bc3a-58173e185047.png)

Reply:
Think of Import/Export-Clixml as psobjeect serialization.  
```powershell
get-date|Export-Clixml -Path .\Date.xml -Encoding utf8

$dt= Import-Clixml -Path .\Date.xml

$dt
Get-Content -Path .\Date.xml

```

![image](https://user-images.githubusercontent.com/3629489/189640172-cf3a7148-a895-4acb-901d-0de53e4d4efa.png)

Reply:
Thank you for trying.